test/swift-files/fill/comment.swift

```diff
@@ -0,0 +1,115 @@
+// swift-mode:test:case-begin (Slash style doc comment)
```

Comment:
Expected and input are separate since it is difficult to transform *from* expected *to* good input. I could not come up with a better way to do this. I am open to suggestions if you prefer another way.